Makefile

```diff
@@ -181,7 +181,7 @@ clean: $(MODEST_BUILD_MODULES_TARGET_CLEAN)
 	rm -f $(call MODEST_LIBRARY_WITH_VERSION) && rm -f $(call MODEST_LIBRARY_STATIC)
 	rm -rf $(TEST_DIR_BASE)
 	$(call MODEST_BUILD_CLEAN_AFTER)
-	rm $(MODEST_PKG_CONFIG_FILE)
+	-rm $(MODEST_PKG_CONFIG_FILE)
```

Comment:
-rm?

Reply:
Meaning don't fail if it's not there.

Reply:
I'm not fan of using `rm -f`

Reply:
Perhaps the above value also needs to lead to a common style?
```sh
 	-rm $(call MODEST_LIBRARY_WITH_VERSION)
        -rm $(call MODEST_LIBRARY_STATIC)
 	-rm -r $(TEST_DIR_BASE)
```

Reply:
Ok, I'll fix them too